server/liveblog/instance_settings/instance_settings.py

```diff
@@ -59,6 +60,13 @@ def is_user_authorized(self):
 
         return flask.g.user.get("is_support", False)
 
+    def get(self, req, lookup):
+        """
+        Retrieve instance settings directly from MongoDB, bypassing the Elasticsearch.
+        This ensures that the most up-to-date configuration is returned to the client.
+        """
+        return self.get_from_mongo(req, lookup)
```

Comment:
we probably don't need this. Instead, we should disable elastic search for this resource.